specs/fr/v1.0.0.md

```diff
@@ -0,0 +1,1052 @@
+----------------
+
+Le bloc de construction principal d'un document TOML est la paire clé/valeur.
+
+Les clés sont à gauche du signe égal et les valeurs sont à droite. Les espaces
+blancs sont ignorés autour des noms et des valeurs de clé. La clé, le signe
+égal et la valeur doivent être sur la même ligne (bien que certaines valeurs
+puissent être réparties sur plusieurs lignes).
+
+```toml
+clé = "valeur"
+```
+
+Les valeurs doivent avoir l'un des types suivants.
+
+- [Chaîne](#chaîne)
+- [Entier](#entier)
+- [Flottant](#flottant)
+- [Booléen](#booléen)
+- [Date-heure avec offset](#date-heure-avec-offset)
+- [Date-heure locale](#date-heure-locale)
+- [Date locale](#date-locale)
+- [Heure locale](#heure-locale)
+- [Tableau](#tableau)
+- [Table inline](#table-inline)
+
+Les valeurs non spécifiées ne sont pas valides.
+
+```toml
+clé = # INVALIDE
+```
+
+Il doit y avoir un saut de ligne (ou EOF) après une paire clé/valeur. (Voir
+[Table inline](#table-inline) pour les exceptions.)
+
+```toml
+first = "Tom" last = "Preston-Werner" # INVALIDE
+```
+
+Clé
+---
+
+Une clé peut être nue, entre guillemets ou pointée.
+
+**Les clés nues** ne peuvent contenir que des lettres ASCII, des chiffres
+ASCII, des traits de soulignement et tirets (`A-Za-z0-9_-`). Notez que les
+clés nues peuvent être composées uniquement de Chiffres ASCII, par ex. `1234`,
+mais sont toujours interprétées comme des chaînes.
+
+```toml
+clé = "valeur"
+bare_key = "valeur"
+clé nue = "valeur"
+1234 = "valeur"
+```
+
+**Les clés entre guillemets** suivent exactement les mêmes règles que les
+chaînes de base ou les littéraux et vous permettent d'utiliser un ensemble
+beaucoup plus large de noms de clés. La meilleure pratique est d'utiliser les
+clés nues sauf en cas d'absolue nécessité.
+
+```toml
+"127.0.0.1" = "valeur"
+"encodage de caractères" = "valeur"
+"ʎǝʞ" = "valeur"
+'clé2' = "valeur"
+'quoted "valeur"' = "valeur"
+```
+
+Une clé nue doit être non vide, mais une clé entre guillemets vide est
+autorisée (bien que découragé).
+
+```toml
+= "pas de nom de clé" # INVALIDE
+"" = "vide" # VALIDE mais déconseillé
+'' = 'vide' # VALIDE mais déconseillé
+```
+
+**Les clés pointées** sont une séquence de clés nues ou entre guillemets
+reliées par un point. Cela permet de regrouper des propriétés similaires :
+
+```toml
+name = "Orange"
+physical.color = "orange"
+physical.shape = "round"
+site."google.com" = true
+```
+
+En JSON, cela vous donnerait la structure suivante :
+
+```json
+{
+  "name": "Orange",
+  "physical": {
+    "color": "orange",
+    "shape": "round"
+  },
+  "site": {
+    "google.com": true
+  }
+}
+```
+
+Pour plus de détails sur les tables définies par les clés pointées,
+reportez-vous à la section [Table](#table) ci-dessous.
+
+Les espaces blancs autour des parties séparées par des points sont ignorés.
+Cependant, la meilleure pratique consiste à ne pas utiliser d'espaces superflus.
+
+```toml
+fruit.name = "banana" # c'est la meilleure pratique
```

Comment:
banane ?

Reply:
Yes, well seen : just one letter diff, hard to find !
Thank you !